src/jit/morph.cpp

```diff
@@ -2066,12 +2066,12 @@ GenTreePtr    Compiler::fgMakeTmpArgNode(unsigned tmpVarNum
 #if FEATURE_MULTIREG_ARGS
 #ifdef _TARGET_ARM64_
```

Comment:
These #ifdefs can be cleaned up now since lvIsMultiregStruct() returns false for all other targets?  The less #ifdef clutter the better it would be.